tldraw/apps/tldraw-logseq/src/styles.css

```diff
@@ -629,6 +629,7 @@ button.tl-select-input-trigger {
   flex-direction: column;
   border-radius: 8px;
   overscroll-behavior: none;
+  margin-left: -40px;
```

Comment:
The design of this may be adjusted by the designer in the future, and it is recommended not to adjust it for now.

Reply:
It is OK.